scripts/docgen-compat/generate-docs.js

```diff
@@ -32,7 +32,7 @@ const { api: apiType, source: sourceFile } = yargs
     type: 'string'
   })
   .option('source', {
-    default: `${repoPath}/packages/firebase/index.d.ts`,
+    default: `${repoPath}/packages/firebase/compat/index.d.ts`,
```

Comment:
This is more of a reminder to myself to audit compat/index.d.ts and make sure it is correct, because I think some  updates were made to the v8 index.d.ts that weren't simultaneously done to compat/index.d.ts.